scripts/S3/generate-s3cfg.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Simple script for generating a sample s3cfg for a couple of known
+# and tested S3 providers.
+
+function usage() {
+    echo "Usage:" 1>&2
+    echo "  $0 aws|exoscale|safespring|citycloud access_key secret_key host_base region" 1>&2
+    exit 1
+}
+
+[ "$#" -lt 4 ] && echo "Too few arguments" && usage
+
+cat <<EOF
+access_key = $2
+secret_key = $3
+host_base = $4
+bucket_location = $5
+EOF
+# Providers that support virtual-hosted-style access to buckets.
+if [ "$1" = "aws" ] || [ "$1" = "exoscale" ]; then
+cat <<EOF
+host_bucket = %(bucket)s.$4
```

Comment:
I wanted to unify the different configrations for different providers so that the only difference is how buckets are addressed. 

Regarding the removal of `use_https`, yea it is enabled by default, see [here](https://github.com/s3tools/s3cmd/blob/v2.0.1/S3/Config.py#L112) and 
```bash
❯ s3cmd --help | grep -A1 -- --ssl
  -s, --ssl             Use HTTPS connection when communicating with S3.
                        (default)
```